tests/wpt/web-platform-tests/resources/testharness.js

```diff
@@ -921,21 +921,23 @@ policies and contribution forms [3].
 
              var p;
              for (p in actual) {
-                 assert(expected.hasOwnProperty(p), "assert_object_equals", description,
+                 assert(p in expected, "assert_object_equals", description,
```

Comment:
@KiChjang you mean my `p in expected` syntax? I actually just tried it and found that it works. It would be good if there is better solution.